tests/test_cache_repo_directories.py

```
@@ -0,0 +1,152 @@
+        return self._payload
+
+
+@pytest.fixture
+def isolated_cache_dir(tmp_path, monkeypatch):
+    monkeypatch.setattr(
+        "platformdirs.user_cache_dir", lambda *_args, **_kwargs: str(tmp_path)
+    )
+    return tmp_path
+
+
+def test_get_repo_directories_caches_with_ttl(monkeypatch, isolated_cache_dir):
```

Ruff (0.14.8)

Unused function argument: `isolated_cache_dir`

(ARG001)